src/executor.test.ts

```diff
@@ -306,31 +302,50 @@ describe('executor', () => {
       ]);
 
       Circuit.runAndCheck(() => {
-        const incomingDelta = executor.flipCoin(
+        const randEncr = Encryption.encrypt([Field(987999)], executorAddress);
+        const randSig = Signature.create(oraclePrivateKey, randEncr.cipherText);
+        [incomingDelta, randomNumber] = executor.flipCoin(
           player1PrivateKey.toPublicKey(),
           merkleMap.get(key),
           witness,
           channelBalance.deltaBalance,
           channelBalance.nonce,
-          channelBalanceSignature
+          channelBalanceSignature,
+          randSig,
+          randEncr.cipherText[0],
+          randEncr.cipherText[1],
+          randEncr.publicKey,
+          executorPrivateKey
         );
-        deltaBalance.add(incomingDelta);
+        console.log(`Incoming Delta Balance: ${incomingDelta.toString()}`);
+        deltaBalance = deltaBalance.add(incomingDelta);
         channelBalance.update(incomingDelta);
+        expect(randomNumber.toString()).toBe('987999');
+        expect(incomingDelta.toString()).toBe('-5');
```

Comment:
Testing one of the largest numbers we can get from the oracle.  It is odd, so balance change should be `-5`